tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Debugger/ProbesTests.cs

```diff
@@ -110,7 +115,9 @@ public async Task LineProbeEmit100SnapshotsTest()
         using var sample = DebuggerTestHelper.StartSample(this, agent, testType.FullName);
         using var logEntryWatcher = new LogEntryWatcher($"{LogFileNamePrefix}{sample.Process.ProcessName}*");
 
-        SetProbeConfiguration(DebuggerTestHelper.CreateProbeDefinition(probes.Select(p => p.Probe).ToArray()));
+        var probeConfiguration = DebuggerTestHelper.CreateProbeDefinition(probes.Select(p => p.Probe).ToArray());
+        SetRcmConfiguration(probeConfiguration, LiveDebugger.Product.Name, EnvironmentHelper.SampleName);
```

Comment:
These two lines will always be repeated together in every test, maybe we should extract out a `SetProbeConfiguration` method?